code/go/0chain.net/smartcontract/benchmark/main/readme.md

```diff
@@ -6,7 +6,37 @@ The blockchain database used in these tests is constructed from the parameters i
 file. Smartcontracts do not (or should not) access tha chain so a populated 
 MPT database is enough to give a realistic benchmark.
 
-To run
+## To run
```

Comment:
Maybe we should add instructions to the main [0chian readme.md](https://github.com/0chain/0chain/blob/staging/README.md), or a link to here.